nixos/modules/services/network-filesystems/litestream.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.litestream;
+in
+{
+  options.services.litestream = {
+    enable = mkEnableOption "litestream";
+
+    package = mkOption {
```

Comment:
Is there any other possible value here?

Reply:
litestream package can be overriden, e.g. `package = pkgs.litestream.override { foo=bar };`

Reply:
I don't see a `foo` flag... https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/database/litestream/default.nix

In fact there isn't really anything to override, is there?

Reply:
`foo` was only an example. Flags may be added in future to litestream, users may want to use custom packages. I've seen this option in many services and find it quite useful.